Feature/desktop styling&api implementation

MagnusTerak opened the pull request:
**Description**

This PR includes desktop styling for ongoing movies & the implementation of API to the cards
---

**Features made in the pull-request**
* Connected API to the card creation through JS
* Styled the cards, chips & searchfield for desktop layout

---

**Changes made in the pull-request**
* Edits in javascript file ongoingMovies.js
* Edits in scss file ongoingMovies.scss
---

**Testing**
This does not interfer with anything else on the website only the styling of OngoingMovies section

I have not done any advanced tests on this code.

---
**Images**
![image](https://github.com/user-attachments/assets/7fdaa389-3ce0-4339-9b22-c1b928f2de23)

Jorlindstrom reviewed (changes requested):
It's fun to see how the design is starting to resemble the Figma design. I think the movie covers are too tall compared to the width. I also think the size of the movie name is too big. It would also be nice if the filtering could align with the width of the movies. You can prepare for further development by setting the movie covers and movie names as a link.

Bekhalm reviewed (commented):
Looks really good and similair to the Figma design. Good choice of colors as well. 

A suggestion is to have less options, to combine one button as 60-talet -80-talet and 90-talet - 10talet. Or possibly one button as a list with all the years.

MagnusTerak commented:
> It's fun to see how the design is starting to resemble the Figma design. I think the movie covers are too tall compared to the width. I also think the size of the movie name is too big. It would also be nice if the filtering could align with the width of the movies. You can prepare for further development by setting the movie covers and movie names as a link.

I've adjusted the height so it doesn't get so long when the width is so small.
I've shrinked the fontsize of the label for the movies. 
I have added a eventListener for clicking on the Image or Label of the movie all data is reachable from the click (ongoingMovies.js) 
If we do the change with the filtering chips as u suggest it will get offcentered because we don't have enough chips to fill the screen, (i tried that in the beginning) That's why i centered them instead to make it look more appropiate

Thank you for your suggestions and your review!

MagnusTerak commented:
> Looks really good and similair to the Figma design. Good choice of colors as well.
> 
> A suggestion is to have less options, to combine one button as 60-talet -80-talet and 90-talet - 10talet. Or possibly one button as a list with all the years.

I understand what u mean with having less options, the only downside i see in this is that you won't be able to select for example 60's 70's and 00's, these are "chips" and u can (later) select multiple options to display results in all chosen chips.
So in my opinion we stick with this and follow the original design on Figma 💯 

Thank you for your review and suggestion!
